Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml.cs

```diff
@@ -195,6 +195,11 @@ private void DrawRectangle(System.Drawing.Point ElementStartPoint, System.Drawin
                 calcHeight = 0 - calcHeight;
             }
             xHighlighterBorder.Height = calcHeight;
+            if (mDriver.GetDevicePlatformType() == eDevicePlatformType.iOS)
+            {
+                xHighlighterBorder.Width = xHighlighterBorder.Width / 1.65;
```

Comment:
Need to see if we need to add also flag for this for doing the changes only for needed devices after testing with iPad and others